client/jetpack-cloud/sections/partner-portal/issue-license-v2/licenses-form/index.tsx

```diff
@@ -149,9 +76,15 @@ export default function LicensesForm( {
 		[ handleSelectBundleLicense ]
 	);
 
-	const onClickIssueLicenses = useCallback( () => {
-		handleShowLicenseOverview();
-	}, [ handleShowLicenseOverview ] );
+	const isSelected = useCallback(
+		( slug: string ) =>
+			selectedLicenses.findIndex(
+				( license ) => license.slug === slug && license.quantity === quantity
+			) !== -1,
```

Comment:
[Non-blocking, Array methods]

Another way to write this, perhaps more idiomatically, could be to use `Array.some`:

```suggestion
			selectedLicenses.some(
				( license ) => license.slug === slug && license.quantity === quantity
			),
```